tools/write_to_sqlite.py

```diff
@@ -0,0 +1,146 @@
+    flux_array = all_entries.reshape(num_blocks, len(bin_widths))
+    return bin_widths, flux_array
+
+def normalize_flux_spectrum(flux_array, bin_widths, num_blocks): #flux spectrum shape
+    norm_flux_array = flux_array.copy()
+    total_flux = np.sum(flux_array, axis=1) #sum over the bin widths of flux array
+
+    for zone_idx in range(num_blocks):
+        norm_flux_array[zone_idx,:] = (norm_flux_array[zone_idx,:] / bin_widths) * (1 / total_flux[zone_idx])  
+    return norm_flux_array, total_flux
+
+def calc_avg_flux_mag(total_flux, active_burn_time):
+    avg_flux = total_flux[0] / active_burn_time
+    # total_flux[0] = total_flux[...]
+    return avg_flux
+
+# def calc_avg_flux_mag(total_flux, num_pulses, pulse_length_list, dwell_time_arr): #avg flux magnitude
+#     #avg = total flux (for each zone) over all energy bins, time-integrated, divided by total irradiation time
+#     # for now, the flux across all zones is assumed to be the same
+#     avg_flux_arr = dwell_time_arr.copy()
+#     for dwell_time_ind, dwell_time in enumerate(dwell_time_arr):
+#         on_time = pulse_length_list[dwell_time_ind] * num_pulses[dwell_time_ind]
+#         off_time = dwell_time * (num_pulses[dwell_time_ind] - 1)
+#         # total_flux[0] = total_flux[...]
+#         avg_flux_arr[dwell_time_ind, :] =  (on_time * total_flux[0] + off_time * 0) / (on_time + off_time)
+#     # (on_time * total_flux[0] (one value for now) + off_time * 0 flux) / (on_time + off_time)
+
+def write_sqlite(adf, inputs, norm_flux_array, avg_flux):
+    #Add new columns to dataframe
+    adf['active_t_irr_(s)'] = inputs['active_burn_time']
+    adf['active_t_irr_(s)'] = aop.convert_times(adf['active_t_irr_(s)'], from_unit='y', to_unit='s')
+    new_columns = {'flux_spectrum_shape' : [norm_flux_array[0]] *  len(adf.index), 
+                    'avg_flux_mag' : avg_flux,
+                    'run_lbl' : [list(inputs.keys())[4]] * len(adf.index)
+                  }
+    adf = adf.assign(**new_columns)
+
+    #Remove some columns:
+    adf.drop(columns=['time', 'time_unit', 'variable', 'var_unit', 'block', 'block_num'], inplace=True)
+
+    #Rename some columns:
+    adf.rename(columns={'value':'num_dens_(atoms/cm3)'}, inplace=True)
+    
+    sqlite_conn = sqlite3.connect('activation_results.db')
+    adf.to_sql('number_densities', sqlite_conn, if_exists='replace', method="multi")
+
+    try:
+        cursor = sqlite_conn.cursor()  
+        sqlite_conn.commit()
+        #cursor.execute("SELECT * FROM number_densities")
+        result = cursor.fetchall()
+
+        cursor.close()
+
+        if sqlite_conn:
+            sqlite_conn.close()
+    except sqlite3.OperationalError as error:
+        print(error)        
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--db_yaml', default = "/filespace/a/asrajendra/research/activationDB/write_sqlite_dir/tools/iter_dt_out.yaml", help="Path (str) to YAML containing inputs")
```

Comment:
I think it makes the most sense for the default to be in the current directory
```suggestion
    parser.add_argument('--db_yaml', default = "iter_dt_out.yaml", help="Path (str) to YAML containing inputs")
```